pkg/controller/plan/kubevirt.go

```diff
@@ -1027,6 +1027,14 @@ func (r *KubeVirt) guestConversionPod(vm *plan.VMStatus, vmVolumes []cnv.Volume,
 				{
 					Name:            "virt-v2v",
 					EnvFrom: []core.EnvFromSource{
+						{
+							Prefix: "V2V_",
+							ConfigMapRef: &core.ConfigMapEnvSource{
+								LocalObjectReference: core.LocalObjectReference{
+									Name: configMap.Name,
+								},
+							},
+						},
```

Comment:
what's the reason for using a config map here - to simplify the code or is there more to it? if there's a good reason for that, does the config map configured with the migration as its owner-reference?

Reply:
You mean why don't we specify the environment variables directly in POD specification instead of using ConfigMap? I suppose we could do that too.